Ui/Component/Listing/Column/SyncedCreditmemo.php

```diff
@@ -71,11 +71,9 @@ public function prepareDataSource(array $dataSource)
                 $creditmemoSyncDate = '';
 
                 try {
-                    $creditmemo = $this->creditmemoRepository->get($item['entity_id']);
-
-                    if ($creditmemo->getTjSalestaxSyncDate()) {
+                    if (isset($item['tj_salestax_sync_date'])) {
                         $creditmemoSyncDate = $this->timezone->formatDate(
-                            new \DateTime($creditmemo->getTjSalestaxSyncDate()),
+                            new \DateTime($item['tj_salestax_sync_date']),
```

Comment:
Is `getTjSalestaxSyncDate()` still being used or can it be removed now?

Reply:
`getTjSalestaxSyncDate()` isn't a defined function and there's nothing to remove (unless I'm misunderstanding).  
https://alanstorm.com/magento-2-magic-getters-and-setters/